### PR TITLE
Hide cell tags view when notebook is closed.

### PR DESCRIPTION
**Pull request by rebornix**
Otherwise we will have a cell tags view in the panel.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,7 +96,7 @@
 					"name": "Cell Tags",
 					"type": "tree",
 					"icon": "$(tag)",
-					"when": "jupyter:showTagsExplorer",
+					"when": "jupyter:showTagsExplorer && jupyter.hasNativeNotebookOrInteractiveWindowOpen",
```
Comment (DonJayamanne) [inline package.json:99]: wouldn't it be better to use some VS Code context keys like `notebookType == jupyter-notebook`
I personally think that would be better

**Review by DonJayamanne — CHANGES_REQUESTED** [review]
Can we try to use some vscode context keys instead of something from jupyter extesion

File: package.json (hunk 1 above)
Comment (rebornix) [reply package.json:99]: `notebookType` is a context key on editor groups which doesn't exist on panel context key tree unfortunately.

**Comment by rebornix** [comment]
@DonJayamanne we currently don't have one in the core unfortunately. I like the idea though, will look into it in debt week.

**Review by DonJayamanne — APPROVED** [review]

**Pull request merged**